TYPO3.TypoScript/Classes/TYPO3/TypoScript/Core/Runtime.php

```diff
@@ -144,6 +144,20 @@ public function injectSettings(array $settings)
     }
 
     /**
+     * Add a tag to the current cache segment
+     *
+     * @param string $key
+     * @param string $value
+     */
+    public function addTag($key, $value)
+    {
+        if (isset($this->settings['enableContentCache'])) {
```

Comment:
huh, why is this check in this direction? "early return if content cache is enabled"?